src/Raven.Studio/wwwroot/App/views/shell/about.html

```diff
@@ -75,13 +75,24 @@ <h2>License information</h2>
                                     <strong data-bind="text: licenseTypeText"></strong>
                                 </div>
                             </div>
-                            <div class="big-label" data-bind="visible: formattedExpiration">
+                            <div class="big-label margin-right margin-right-sm" data-bind="visible: formattedExpiration">
                                 <div class="content">
                                     <small data-bind="text: expiresText"></small>
                                     <strong data-bind="html: formattedExpiration"></strong>
                                     <small data-bind="text: automaticRenewText"></small>
                                 </div>
                             </div>
+                            <div class="big-label">
+                                <div class="content">
+                                    <small>License Connectivity
```

Comment:
I dont quite like this name. We don't connect license - we check if license server can be reached.

Reply:
![image](https://user-images.githubusercontent.com/1250431/139747669-d753d9c9-bf1d-4a78-9923-42bc7830653f.png)

Reply:
![image](https://user-images.githubusercontent.com/1250431/139747707-2513c16f-ec4d-47e5-861e-9728783b76e2.png)

Reply:
Maybe let's call this: `License Server` with states:
No connection (red with icon + tooltip)
Valid connection - green , w/o tooltip?

Reply:
done